mobile/src/reducers/organization/feed.model.ts

```diff
@@ -0,0 +1,27 @@
+import {dataMember, required} from 'santee-dcts';
+import { Employee } from './employee.model';
+
+export class Feed {
+
+    @dataMember()
+    @required()
+    public messageId: string;
+
+    @dataMember()
+    @required()
+    public employeeId: string;
+
+    @dataMember()
+    @required()
+    public title: string;
+
+    @dataMember()
+    @required()
+    public text: string;
+
+    @dataMember()
+    @required()
+    public datePosted: string;
+
+    public employee: Employee;
```

Comment:
where is it used?

Reply:
I mean employee field

Reply:
[home-feeds-screen.tsx:19](https://github.com/SoftwareCountry/arcadian-assistant/pull/37/files#diff-151d03fc0a5c35559a55bae1e6145052R19)